wire-ios-data-model/Source/Model/Conversation/ZMConversation+Message.swift

```diff
@@ -384,11 +384,7 @@ extension ZMConversation {
         }
 
         message.sender = ZMUser.selfUser(in: moc)
-
-        if expires {
-            let expirationDate = message.setExpirationDate()
-            WireLogger.messaging.debug("set expiration date \(expirationDate)", attributes: logAttributes)
-        }
+        message.shouldExpire = expires
```

Comment:
@markusfassbender should we toggle shouldExpire again once it's sent ?
What's the default value of shouldExpire?

Reply:
Good points:
- the default value is `false` and defined this way (and non-optional) in the CoreData model.
- in my opinion it is not required to reset the `shouldExpire` as it is just a flag to let the other piece of code know when a timer should be started. @johnxnguyen what do you think?

Reply:
I agree with @markusfassbender, once it's sent it becomes irrelevant.